src/apis/instance.ts

```diff
@@ -16,3 +16,17 @@ Instance.interceptors.request.use((config) => {
 
   return config
 })
+
+Instance.interceptors.response.use(
+  function (response) {
+    return response
+  },
+
+  async function (error) {
+    const errorDataStatus = error.response.status
+
+    if (errorDataStatus >= 500 && errorDataStatus < 600) {
```

Comment:
```suggestion
    if (errorDataStatus >= 400 && errorDataStatus < 600) {
```
https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-2-7/assets/93248349/aa8c3591-7b55-438a-9c35-8b34b7e55b72

저도 서버 에러를 테스트 해볼 방법이 없어서 일부러 400 에러를 내게 끔 만들고 테스트 해봤는데..
useNavigate가 아니라 location.href로 조작해서 그런건지 (저도 확실치 않습니다만) 제대로 에러 페이지로 가지는 않는것 같습니다.

그리고 개인적인 의견으로는 서버 에러 발생시에도 기존 IssueList의 에러 UI로 처리 가능하므로, 서버 에러 페이지는 없어도 될 것 같습니다

Reply:
404는 api 통신을 통해 일어난 에러가 아니라 interceptors에서 걸리지 않은게 아닐까 라는 생각을 가지고 있습니다 ..!

이번 pr은 이슈 상세 페이지를 제가 구현했는데, 상세페이지의 에러는 어떻게 처리할지 생각하다가 만든 pr이었어요 !
그래서 API 통신을 할 때 interceptors를 사용하면 전역적으로 사용할 수 있을 것 같아 해당 코드를 적용했는데 이미 이슈 리스트에는 따로 이슈 UI가 있었군요 ..!

그럼 상세 페이지에도 따로 에러 UI를 만드는 게 나을까요 ?

Reply:
제가 이슈 리스트쪽 에러 UI를 빼먹어서 스타일링 하려 봤는데, 이게 한번 등장하면 닫을수 있는 형태로 구현을 해야해서, 
어차피 닫기가 있는 유형이라면, 둘다  이 페이지로 같이 빼는것이 저는 더 괜찮을거 같다고 생각했어요!

Reply:
이슈 페이지의 에러 시 UI는 이슈 조회 에러를 처리한 부분인데 500번대 서버에러를 위한 컴포넌트는 아니었습니다
제 생각에는 페이지보다 컴포넌트로 하나 빼주시면 여기저기 필요한 곳에서 쓸 수 있을 것 같아요

이슈에서도 컴포넌트로 조건 분기를 했거든요
`{isError && <IssueListError />}`

Reply:
> 이번 pr은 이슈 상세 페이지를 제가 구현했는데, 상세페이지의 에러는 어떻게 처리할지 생각하다가 만든 pr이었어요 ! 그래서 API 통신을 할 때 interceptors를 사용하면 전역적으로 사용할 수 있을 것 같아 해당 코드를 적용했는데 이미 이슈 리스트에는 따로 에러 UI가 있었군요 ..!

@wisdomin121  이슈 상세 페이지만 적용되는 에러 페이지 였군요 😅

> 제가 이슈 리스트쪽 에러 UI를 빼먹어서 스타일링 하려 봤는데, 이게 한번 등장하면 닫을수 있는 형태로 구현을 해야해서,
어차피 닫기가 있는 유형이라면, 둘다 이 페이지로 같이 빼는것이 저는 더 괜찮을거 같다고 생각했어요!

![이미지](https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-2-7/assets/93248349/0440738b-21b4-41bc-b646-b35dfd642d30)

@JangHyunjeong 혹시 이런 느낌일까요?

> 음 하지만.. 모르겠어요 .. 페이지가 좋을까요?

@anyl92  저도 컴포넌트가 좋을 것 같습니다. 페이지로 빼버리면 리다이렉트 될 때 이슈 리스트의 경우는 기존에 불러왔던 데이터도 못 보게 되어서 불편할 것 같아요.

Reply:
@anyl92  @salmontaker 네! 그런 팝업같은 형태요!  오.. 리다이렉트 이슈까지 생각하면.. 컴포넌트가 더 나을거같네요!

Reply:
@salmontaker 앗 상세 페이지만은 아니고 전체에 적용이 되긴 하는데, 상세 페이지만을 생각하고 만들었다는 의미였습니다 ..!

@salmontaker @JangHyunjeong @anyl92 그러면 리스트페이지의 에러는 전에 있던 그대로 유지하고, 상세페이지의 에러는 들어가기 전에 모달같은 걸로 띄우는 걸까요 ?!

Reply:
@wisdomin121 
네네 말씀하신대로 사진 처럼 생각했었는데... 
그런데.. 다시 생각해보니까 리스트는 이전의 데이터가 유지되었으면.. 하는거고.. 
상세는 데이터를 불러오지 않아서... 이전 데이터 유지를 안해도 되니까 굳이.. 팝업으로 뺄 필요가 없겠네요!  
저는 상세는 이대로 사용해도 좋을거 같네요! 
리스트는 제가 팝업 형태로 스타일 따로 진행하고요

![image](https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-2-7/assets/85441226/848513ad-2822-4e42-932c-a87f46153b6d)